src/index.html

```diff
@@ -11,6 +11,36 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://VLabovskyi.github.io/layout_search-bar-airbnb/"
+      method="post"
+      data-qa="big"
+      class="form"
+    >
+      <label>
+        <input
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;"
+          data-qa="keypress"
+          class="search searchBig"
```

Comment:
always try to write classes in kebab-case or BEM
```suggestion
          class="search search-big"
```